File: Fabric/src/main/java/gay/object/hexdebug/fabric/HexDebugClientFabric.java
package gay.object.hexdebug.fabric;

import net.fabricmc.api.ClientModInitializer;
import gay.object.hexdebug.HexDebugClient;

/**
 * Fabric client loading entrypoint.
 */
public class HexDebugClientFabric implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        HexDebugClient.init();
    }
}

File: Fabric/src/main/kotlin/gay/object/hexdebug/fabric/HexDebugClientFabric.kt
package gay.`object`.hexdebug.fabric

import gay.`object`.hexdebug.HexDebugClient
import net.fabricmc.api.ClientModInitializer

object HexDebugClientFabric : ClientModInitializer {
    override fun onInitializeClient() {
        HexDebugClient.init()
    }
}
